md/getting-started-tutorial.md

```diff
@@ -486,34 +466,39 @@ Set the **Level** property to Level 4, so it is the same size as the title of th
 
     | Property  | Value  |
     | --------- | ------ |
-    | Hidden  | `formInput.status == 'approved'` (click on "bind" icon) |
+    | Hidden  | `formInput.status === 'approved'` (first click on "fx"/"bind" icon) |
     | Label  | Refusal reason  |
     | Value  | `formInput.refusalReason`  |
+    | Required | yes |
 
-    The dynamic _Hidden_ property will hide the widget when the request status is set to 'approved'.
+    The dynamic _Hidden_ property will hide the widget when the request status is set to 'approved'.  
     Otherwise, the field will be marked as required and the form validation will prevent the user from refusing a request without a reason.
 12. Remove the Status widget that was added automatically.
 13. Add a Radio buttons widget with the following properties:
 
     | Property  | Value  |
     | --------- | ------ |
     | Label  | Review decision  |
-    | Available values (click on the binding icon next to the value) | statusValues  |
+    | Available values (click on the binding "fx" icon next to the value first) | statusValues  |
     | Displayed keys  | label  |
     | Returned keys  | value  |
     | Selected Value  | `formInput.status`  |
 
-14. Select the _Submit_ widget and bind its _Disabled_ property to `$form.$invalid`
-15. Preview the form to test the validation and rearrange the widgets until it looks like this in the editor:
+14. Preview the form to test the validation and rearrange the widgets until it looks like this in the editor:
 
   ![Process instantiation form](images/images-6_0/7.2-getting-started-review-form.png)
 
 ## Run process
 
 You can now run the process using the forms that you created. Make sure that you have saved everything in the UI Designer. Then click **_Run_** in the Studio coolbar.
 
-Fill out the request form as Helen, then log in as William and do the review task.    
-When the review form is displayed, it contains the information that Helen entered in the request form. You can approve or refuse the request.
+Fill out the request form as Walter, then log in as Helen and do the review task.  
+When the review form is displayed, it contains the information that Walter entered in the request form. You can approve or refuse the request.
+
+Once process is finished you can see the request information as stored in the test database in Bonita Studio:
```

Comment:
Is "test database" a term we use in the rest of the documentation ?
I would rather say "development database", but I am not sure